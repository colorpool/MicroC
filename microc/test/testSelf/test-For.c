void main(int n) {
    int i;
    print 'd' : n;
    for( i = 0 ; i < 10;  i = i + 1){
        n=n-1;
        print 'd' : n;
    }
    print 'd' : n+1;
}