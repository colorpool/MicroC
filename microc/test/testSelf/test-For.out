24 19 1 5 25 15 1 13 11 22 15 -1 13 0 1 1 0 0 12 15 -1 16 52 13 13 11 0 1 2 12 15 -1 13 11 22 15 -1 13 0 1 1 13 0 1 1 11 0 1 1 12 15 -1 13 0 1 1 11 0 10 7 18 23 13 11 0 1 1 22 21 2